.ruby-version

```diff
@@ -1 +1 @@
-2.2.2
\ No newline at end of file
+2.3.0
```

Comment:
aye!

On Tue, Jun 21, 2016 at 3:18 PM, Cyril Kyburz <notifications@github.com>
wrote:

> In .ruby-version
> <https://github.com/renuo/renuo-upload-signing/pull/3#discussion_r67865493>
> :
>
> > \ No newline at end of file
> > +2.3.0
>
> will be in the next release
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/renuo/renuo-upload-signing/pull/3/files/981d50b948cc2e503e4f5e952c81dfa25cbeafff#r67865493>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAT9Fx2jtIKwW5pSU5O7gzTkzdu5SHPRks5qN-SzgaJpZM4I6jb3>
> .
>



-- 
*Renuo GmbH*
Lukas Elmer
Founder
MSc ETH in Computer Science
BSc HSR in Computer Science

Industriestrasse 44
8304 Wallisellen

Mail lukas.elmer@renuo.ch
Web https://www.renuo.ch

Sollten Sie diese E-Mail irrtümlich erhalten haben, so ist der gesamte
Inhalt (inkl. Anhang) unverzüglich zu löschen. Bitte benachrichtigen Sie in
diesem Fall Renuo GmbH umgehend.